src/cli/user.js

```diff
@@ -0,0 +1,310 @@
+'use strict';
+
+const { Command, Option } = require('commander');
+
+module.exports = () => {
+	const userCmd = new Command('user')
+		.description('Manage users')
+		.arguments('[command]');
+
+	userCmd.configureHelp(require('./colors'));
+	const userCommands = UserCommands();
+
+	userCmd
+		.command('info')
+		.description('Display user info by uid/username/userslug.')
+		.option('-i, --uid <uid>', 'Retrieve user by uid')
+		.option('-u, --username <username>', 'Retrieve user by username')
+		.option('-s, --userslug <userslug>', 'Retrieve user by userslug')
+		.action((...args) => execute(userCommands.info, args));
+	userCmd
+		.command('create')
+		.description('Create a new user.')
+		.arguments('<username>')
+		.option('-p, --password <password>', 'Set a new password. (Auto-generates if omitted)')
+		.option('-e, --email <email>', 'Associate with an email.')
+		.action((...args) => execute(userCommands.create, args));
+	userCmd
+		.command('reset')
+		.description('Reset a user\'s password or send a password reset email.')
+		.arguments('<uid>')
+		.option('-p, --password <password>', 'Set a new password. (Auto-generates if passed empty)', false)
+		.option('-s, --send-reset-email', 'Send a password reset email.', false)
+		.action((...args) => execute(userCommands.reset, args));
+	userCmd
+		.command('delete')
+		.description('Delete user(s) and/or their content')
+		.arguments('<uids...>')
+		.addOption(
+			new Option('-c, --content [operation]', 'Delete user content, leave content, or delete content only')
+				.choices(['delete', 'leave', 'content-only']).default('delete')
```

Comment:
Hm, this is confusing. I cannot wrap my head around the options at first read (also, you have "delete content" in there twice.

I am assuming:

* `delete`: Deletes account **and** content
* `leave`: Deletes account only
* `content-only`: Deletes content only

Can we use the nomenclature used in `public/src/modules/accounts/delete.js`? `account`, `content`, `purge`

Reply:
Done :+1: